assets/cubyz/sbb/tree/oak/1/leaf_1.zig.zon

```diff
@@ -0,0 +1,4 @@
+.{
+	.blueprint = "cubyz:tree/oak/1/leaf_1",
+	.children = .{},
+}
```

Comment:
@Argmaster could we maybe allow adding end nodes without having to specify an empty file like this?

Reply:
Or do you mean like without having SBB file at all?
Maybe, it would mean I would have to allow inlined SBB definitions I will have to dive deeper to see what is the cost.

Reply:
https://github.com/PixelGuys/Cubyz/issues/1403